dropwizard-metrics-influxdb/src/main/resources/META-INF/services/com.izettle.metrics.dw.tags.Transformer

```diff
@@ -0,0 +1,3 @@
+com.izettle.metrics.dw.tags.ClassBasedTransformer
```

Comment:
I might miss something, but why do we need this? I meant adding `com.izettle.metrics.dw.tags.Transformer` file in `META-INF`.

Reply:
It's needed for resolving the names (`@JsonTypeName`) using the [DiscoverableSubtypeResolver](https://github.com/dropwizard/dropwizard/blob/master/dropwizard-jackson/src/main/java/io/dropwizard/jackson/DiscoverableSubtypeResolver.java).

Reply:
Ah, cool, thanks. 👍